### DISCUSSION TITLE
PR如何通过测试和review？

### DISCUSSION THREAD

**Discussion by zzyyu** [element-plus/element-plus#12627] [Q&A]
#12559 PR上显示还有3项测试待批准，请问如何推进这些流程？
![image](https://user-images.githubusercontent.com/33147972/234746180-4b284296-d0e4-4912-aaac-0dda010847df.png)

**Comment by yj-liuzepeng** [comment]
我也想知道

**Comment by Altermoe** [comment]
要等，等有 review 权限的项目参与者有空了就能给你 review 了。

**Comment by ishowman** [comment]
组件源码更改后，怎样自动生成对应的类型声明呢？  @Altermoe @zzyyu

**Reply by Altermoe** [reply]
不知道你是不是在找这个：[vite-plugin-dts](https://github.com/qmhc/vite-plugin-dts)
